src/utils/index.ts

```diff
@@ -34,16 +34,29 @@ export function fileFormatToString(extension: FileFormat | string): string {
 }
 
 /**
- * Deletes a directory and all subdirectories
+ * Creates a directory and all subdirectories synchronously
  *
  * @param {any} dirPath The directory's path
  */
-export function deleteDirectoryRecursively(dirPath: string): void {
+export function createDirectoryRecursivelySync(dirPath: string): void {
```

Comment:
I guess you've used the node convention when naming this function. 

I prefer naming the async functions with the `async` suffix a la .NET. I think it's clearer as normally the majority of the functions (not the ones belonging to node.js) are syncrhonous.